insights/config/specs.py

```diff
@@ -437,6 +437,9 @@
     "vsftpd"                    : SimpleFileSpec("etc/pam.d/vsftpd"),
     "vsftpd.conf"               : SimpleFileSpec("etc/vsftpd/vsftpd.conf"),
     "woopsie"                   : CommandSpec(r"/usr/bin/find /var/crash /var/tmp -path '*.reports-*/whoopsie-report'"),
+    "x86_pti_ibpb_ibrs"         : All([SimpleFileSpec("sys/kernel/debug/x86/pti_enabled"),
+                                    SimpleFileSpec("sys/kernel/debug/x86/ibpb_enabled"),
+                                    SimpleFileSpec("sys/kernel/debug/x86/ibrs_enabled")]),
```

Comment:
@kgrant-rh would you mind to change this to be consistent with your 1.x PR #882, thanks.